src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public enum Colors {
+    Black("Black"),
+    White("White"),
+    Red("Red"),
+    Yellow("Yellow"),
+    Orange("Orange"),
+    Pink("Pink"),
+    Green("Green"),
+    Blue("Blue");
```

Comment:
Don’t forget how to name enums according to style guide (the same with naming of constant values)
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names